config/locales/sv.yml

```diff
@@ -0,0 +1,18 @@
+sv:
+  header:
+    main: VÄLKOMMEN!
+    second: ◊ Sprid nyheterna ◊
+  signup:
+    main: Registrera
+  newarticle:
+    main: Ny Artikel
+  signin:
+    main: Logga in
+  logout:
+    main: Logga ut
+  pay:
+    main: Kortbetalning
+  localarticles:
+    main: Lokala Artiklar
+  allarticles:
```

Comment:
`allaarticles`?  Mixing languages in a key name do not seem right. Also, please use camel_case.